src/decision/dMCDA.jl

```diff
@@ -45,6 +49,53 @@ function mcda_methods()
     ]
 end
 
+"""
+    mcda_method_names()::Vector{String}
+
+List of MCDA method names.
+"""
+function mcda_method_names()::Vector{String}
+    return [
+        "Cocoso",
+        "Mairca",
+        "Moora",
+        "Piv",
+        "Vikor"
+    ]
+end
+
+"""
+    mcda_method_encoding(mcda_name::String)::Union{Int64, Nothing}
+
+Find the scenario encoding of the given mcda method. Throws `ArgumentError` if method not 
+found.
+"""
+function mcda_method_encoding(mcda_name::String)::Int64
+    method_idx = findfirst(mcda_method_names() .== mcda_name)
```

Comment:
```suggestion
    method_idx = findfirst(mcda_method_names() .== titlecase(mcda_name))
```

This way the user can specify "PIV", "CoCoSo", "MOORA" etc and it will still match

Reply:
If my suggestion further above is adopted, then:

```suggestion
    method_idx = findfirst(mcda_method_names() .== uppercase(mcda_name))
```